deb-get

```diff
@@ -357,7 +357,7 @@ function validate_deb() {
     elif  [ -n "${PPA}" ]; then
         METHOD="ppa"
     else
-        if (([ "${METHOD}" = github ] && [ -e "${CACHE_DIR}/${APP}.json" ]) || ([ "${METHOD}" != github ] && [ "${ACTION}" != "prettylist" ])) && ([ -z "${URL}" ] || [ -z "${VERSION_PUBLISHED}" ]); then
+        if (([ "${METHOD}" = github ] && [ -e "${CACHE_DIR}/${APP}.json" ]) || [ "${METHOD}" != github ]) && [ "${ACTION}" != "prettylist" ] && ([ -z "${URL}" ] || [ -z "${VERSION_PUBLISHED}" ]); then
```

Comment:
It's 3am and my mother tongue is SQL so I might be confused or old-fashioned, but I'm pretty sure you should use "{ ..; }"  parenthesis to group those tests rather than "(..)" if only for performance reasons.
https://wiki.bash-hackers.org/commands/classictest#precedence_and_parenthesis

Reply:
Can confirm this fixes `prettylist` - thus cog of README.
Need more time and coffee to apply brain to logic without crashing.

Reply:
> I'm pretty sure you should use "{ ..; }" parenthesis to group those tests rather than "(..)"

Indeed, I did not connect the fact that the parentheses I used to mark precedence are actually sub-shells.